### ISSUE TITLE
Button to DeltaAxis (mouse)

### ISSUE THREAD

**Issue by TiKevin83** [Snoothy/UCR#116]
Is there a pipeline for buttons as mouse movement?  I'm using Extrems' GBA-as-gc-controller in combination with vjoy GC controller adapter drivers to use a GBA SP as a vjoy controller.  I was able to map A and B to left and right click with UCR and Intersection, but I also am interested if I can translate dpad to mouse movement (DeltaAxis in UCR-Plugins?)  is there a current solution for buttons to DeltaAxis?

**Comment by evilC** [comment]
No, not currently, but it would not be too hard to make a basic version - I guess you would need, at a minimum, two button inputs and one delta output, plus an integer value for how much movement to send

**Comment by Xyaren** [comment]
Maybe allow for increase of movement over time while pressing the button
